sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/ViewFn.java

```diff
@@ -36,6 +39,7 @@
  * @param <PrimitiveViewT> the type of the underlying primitive view, provided by the runner
  *        {@code <ViewT>} the type of the value(s) accessible via this {@link PCollectionView}
  */
+@Internal
```

Comment:
I think things that are `@Internal` trump `@Experimental`. It means users just can't use them at all without being broken.